cmd/notify-http.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * Minio Cloud Storage, (C) 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+type httpNotify struct {
+	Enable bool   `json:"enable"`
+	Addr   string `json:"address"`
+}
+
+func newHTTPNotify(accountID string) (*logrus.Logger, error) {
+	rNotify := serverConfig.GetHTTPNotifyByID(accountID)
+	fmt.Println(rNotify)
+
```

Comment:
I don't think all light-weight servers provide CONNECT,OPTIONS etc so I have implemented the dial check and then at runtime if the post fails it will return an error.